.github/workflows/stg-workflow.yml

```
@@ -0,0 +1,108 @@
+      - name: Run ESLint
+        run: pnpm lint
+
+      - name: Build
+        run: make build
+
+  create-release-pr:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/dev'
+    needs: [main-api, fetcher, nextjs-web]
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)